libc/config/windows/headers.txt

```diff
@@ -0,0 +1,32 @@
+set(TARGET_PUBLIC_HEADERS
+    libc.include.assert
+    libc.include.ctype
+    libc.include.dirent
+    libc.include.errno
+    libc.include.features
+    libc.include.fenv
+    libc.include.float
+    libc.include.inttypes
+    libc.include.limits
+    libc.include.link
```

Comment:
probably don't want the linux specific link.h for windows.  I think the rest are fine.